ebpfapi/Source.def

```diff
@@ -79,6 +79,7 @@ EXPORTS
     bpf_program__attach_xdp
     bpf_program__autoload
     bpf_program__fd
+    bpf_program__flags
```

Comment:
This PR needs to also update include\bpf\libbpf.h to add the doxygen for `bpf_program__flags` and `bpf_program__set_flags`.  Setting "Request Changes" due to this gap.

Reply:
Done.